Dockerfile.android

```
@@ -0,0 +1,187 @@
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    wget unzip xz-utils ca-certificates \
+    cmake ninja-build make git \
+    ruby ruby-getoptlong perl python3 rsync file cpio \
+    lsb-release software-properties-common gnupg \
+    && rm -rf /var/lib/apt/lists/*
+
+# Host clang (same version Bun uses) — we cross-compile via --target/--sysroot,
+# not via the NDK's bundled clang. apt.llvm.org installs version-suffixed names
+# only (ld.lld-21, not ld.lld), so add unversioned links for -fuse-ld=lld.
+RUN wget -qO- https://apt.llvm.org/llvm.sh | bash -s -- ${LLVM_VERSION} && \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)